### PR TITLE
TST-521 - API endpoint

### PR DESCRIPTION
**Pull request by wbyrnetx**

### CODE REVIEW

**Review by kellyhines57 — APPROVED** [review]

File: config.json [hunk 1]
```diff
@@ -409,7 +409,7 @@
       "images": {},
       "settings": {
         "ts_tsapi_base_url": "https://qa1-tsapi.tastefullysimple.com",
-        "ts_tsapi_host_party_base_url": "https://qa1-party.api.tastefullysimple.com",
+        "ts_tsapi_host_party_base_url": "https://party.api.tastefullysimple.com",
```
Comment (TastefullySimple) [inline config.json:412]: This change should be reverted.

File: config.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
     "ts_current_store_base_url": "https://tastefully-simple-sandbox-2.mybigcommerce.com",
     "ts_shop_more_url": "/shop/",
     "ts_tsapi_base_url": "https://qa1-tsapi.tastefullysimple.com",
-    "ts_tsapi_host_party_base_url": "https://qa1-party.api.tastefullysimple.com",
+    "ts_tsapi_host_party_base_url": "https://party.api.tastefullysimple.com",
```
Comment (TastefullySimple) [inline config.json:51]: This change should be reverted.

File: config.json [hunk 3]
```diff
@@ -385,7 +385,7 @@
       "images": {},
       "settings": {
         "ts_tsapi_base_url": "https://qa1-tsapi.tastefullysimple.com",
-        "ts_tsapi_host_party_base_url": "https://qa1-party.api.tastefullysimple.com"
+        "ts_tsapi_host_party_base_url": "https://party.api.tastefullysimple.com"
```
Comment (TastefullySimple) [inline config.json:388]: This change should be reverted.

File: config.json [hunk 4]
```diff
@@ -436,7 +436,7 @@
         "ts_current_store_base_url": "https://tastefully-simple-sandbox-1.mybigcommerce.com",
         "ts_shop_more_url": "/shop/",
         "ts_tsapi_base_url": "https://stage-tsapi.tastefullysimple.com",
-        "ts_tsapi_host_party_base_url": "https://qa1-party.api.tastefullysimple.com",
+        "ts_tsapi_host_party_base_url": "https://party.api.tastefullysimple.com",
```
Comment (TastefullySimple) [inline config.json:439]: This should be https://stage.api.tastefullysimple.com

**Pull request merged**